pandas/core/ops.py

```diff
@@ -255,8 +255,10 @@ def _get_frame_op_default_axis(name):
 ----------
 other : Series or scalar value
 fill_value : None or float value, default None (NaN)
-    Fill missing (NaN) values with this value. If both Series are
-    missing, the result will be missing
+    Fill existing missing (NaN) values, and any new element needed for
```

Comment:
I would remove the 'and any new elemnt needed for successful array alignment', this is redundant with your last sentence.

Reply:
I'm not sure which sentence are you referring to. The "and any new element needed" sentence refers to the alignment process. The last sentence ("If data in both corresponding...") only deals with a "corner case" of two NaNs, without any direct reference to the data alignment process.

Reply:
don't use the word 'array' as these are not arrays. this is still not very clear.

Reply:
I like this phrasing much better. Could you change "new element" to "new missing values"

You can remove the commas around the "and any new element... " clause.